pkg/ruler/metrics.go

```diff
@@ -0,0 +1,36 @@
+package ruler
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var samplesEvicted = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
I would love to see these instantiated rather than globally defined (example: https://github.com/grafana/loki/blob/main/pkg/ingester/metrics.go)